src/modules/doctor/doctor.service.ts

```diff
@@ -59,6 +59,19 @@ export default class DoctorService {
     }
   }
 
+  async getDoctorByEmail(email: string): Promise<Doctor | null> {
+    try {
+      const doctor = await this.doctorRepository
```

Comment:
no we use query builder only

Reply:
deleted (the same issue you write in first comment)